src/utils/useRoutedTabs.js

```diff
@@ -7,15 +7,28 @@ import { useHistory, useLocation } from 'react-router-dom'
 const useRoutedTabs = (tabPaths, defaultTabIndex = 0) => {
   const { pathname } = useLocation()
   const history = useHistory()
+
+  // get breadcrumbs
+  const segments = pathname.split('/')
+
   const [tabIndex, setTabIndex] = useState(() => {
     // Set the initial tabIndex from the URL on page load
-    const matchedIndex = tabPaths.indexOf(pathname)
+    const matchedIndex = tabPaths.findIndex((tabName) =>
+      pathname.includes(tabName),
```

Comment:
I'd actually do `pathname.startsWith(tabName)`.

Reply:
Actually Siraj, consider using `matchPath` from React router to handle this. https://reactrouter.com/web/api/matchPath

`exact: false, strict: false` (which are already defaults)